install.go

```diff
@@ -160,7 +158,7 @@ func install(parser *arguments) error {
 	}
 
 	if len(do.Aur) == 0 && len(arguments.targets) == 0 && (!parser.existsArg("u", "sysupgrade") || mode == ModeAUR) {
-		fmt.Println("There is nothing to do")
```

Comment:
Should I make some variable or function for unifying this output or this duplicating is normal?

Reply:
This only shows up twice right? https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)

Reply:
Good article, thanks! I'm personally have something like "Rule of two" (**dup**licate, like **two**), but arguments for "three" make sense.

Reply:
Of course it's just a rule of thumb so don't follow it to closely. If only I could `#define` a return on error :(